docs/networking.md

```diff
@@ -350,6 +351,77 @@ $ kops create cluster \
 
 In case of any issues the directory `/var/log/aws-routed-eni` contains the log files of the CNI plugin. This directory is located in all the nodes in the cluster.
 
+### Cilium Example for CNI and Network Policy
+
+Cilium is open source software for transparently securing the network connectivity between application services deployed using Linux container management platforms like Docker and Kubernetes.
+
+#### Installing Cilium on a new Cluster
+
+The following command sets up a cluster, in HA mode, with Cilium as the CNI and networking policy provider
+
+```console
+$ export ZONES=mylistofzones
+$ kops create cluster \
+  --zones $ZONES \
+  --master-zones $ZONES \
+  --master-size m4.large \
+  --node-size m4.large \
+  --networking cilium\
+  --yes \
+  --name myclustername.mydns.io
```

Comment:
For the future, we try to stick with `example.com` - it's actually reserved for the purpose: https://www.iana.org/domains/reserved